vhost-user-backend/src/handler.rs

```diff
@@ -574,6 +575,17 @@ where
             .map_err(VhostUserError::ReqHandlerError)
     }
 
+    fn get_shared_object(&mut self, uuid: VhostUserSharedMsg) -> VhostUserResult<File> {
+        match self.backend.get_shared_object(uuid) {
+            Ok(Some(shared_file)) => Ok(shared_file),
+            Ok(None) => Err(VhostUserError::IncorrectFds),
```

Comment:
We are mapping `None` on an VhostUserError, so can we just return `Result<File>` from `VhostUserBackend::get_shared_object` ?